students_documentation/summary_notebook/summary.qmd

```diff
@@ -0,0 +1,76 @@
+# Added Features to CounterfactualExplanations.jl
+
+In this notebook, we document the new features that we have added to the CounterfactualExplanations.jl package. These features include increasing the scope of predictive models that are compatible with the package, adding unit testing and documentation, and improving the package infrastructure. We hope that this notebook will serve as a useful reference for both ourselves and the course staff.
```

Comment:
I would rephrase the second sentence as "These features include adding new counterfactual generators into the package, increasing the scope of predictive models that are compatible with the package, adding unit testing and documentation, and improving the package infrastructure," since adding generators is a very core part of what we're doing. This is a minor detail, though, and everything else looks great, so I'll approve.